### DISCUSSION TITLE
note editor layout messed up

### DISCUSSION THREAD

**Discussion by horvatkm** [TriliumNext/Notes#592] [General]
Hi,
I'm not sure what I've done wrong, but for some reason layout of note editor somehow got messed up. What should I do to fix it? App restart doesn't help. Not even reinstall helps.

 I'm running Windows 10 with latest TrilliumNext Notes 0.90.8. 

![trillium_messed_up](https://github.com/user-attachments/assets/eb828426-6c7f-49f6-9e89-72bc265eb563)
![image](https://github.com/user-attachments/assets/79b32541-1776-4cd0-b553-ed9e8524c23f)

**Comment by Nriver** [comment]
Open Trilum and do these step by step. Blind man working : )

- `alt+o` 
- `ctrl+.`
- `ctrl+.` yes, hit twice
- `enter`
- `tab`
- `ctrl+a`
- `ctrl+v` paste these.
```
UPDATE "options" SET "value" = '9999' WHERE "name" = 'maxContentWidth';
```
- `ctrl + enter` to excute.
- Restart trilium, you should be fine.

**Reply by horvatkm** [reply]
Thanks, I must have somehow set maxContentWidth to zero. Allowing user to set any value to maxContentWidth  and not giving him any warning that he may make all his notes inaccessible seems to me like a bug.

**Reply by Nriver** [reply]
Yeah, I agree that there should be a warning message or a lowest value limit. @eliandoran

**Reply by perfectra1n** [reply]
@Nriver MVP award 🥇

**Comment by eliandoran** [comment]
To be handled in #593 .
